src/Tribe/Admin/Notice/Archive_Slug_Conflict.php

```diff
@@ -1,101 +0,0 @@
-<?php
-
-
-/**
- * Class Tribe__Admin__Notice__Archive_Slug_Conflict
- *
- * Takes care of adding an admin notice if a page with the `/events` slug has been created in the site.
- */
-class Tribe__Admin__Notice__Archive_Slug_Conflict {
```

Comment:
I know it would seem very unlikely that third parties are using this class - but should we deprecate? I'm not saying we must, just asking the question :wink: